examples/index.html

```diff
@@ -25,6 +25,7 @@
 
 {# {%- from "invenio_pidrelations/macros/pid_versions.html" import pid_versions_dropdown -%} #}
 <!DOCTYPE html>
+{%- from "invenio_pidrelations/macros/pid_versions.html" import pid_versions_dropdown -%}
```

Comment:
This is not used in the example, although it's Bootstrap specific, and thus might be hard to integrate without including `invenio-theme`. I might find a place in case a dedicated `/record/:pid` endpoint is added to the example app..